ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__azure-4.16-nightly-x86.yaml

```diff
@@ -91,8 +91,8 @@ tests:
     cluster_profile: azure-qe
     env:
       BASE_DOMAIN: qe.azure.devcluster.openshift.com
-      COMPUTE_NODE_TYPE: Standard_D8s_v3
-      OPENSHIFT_INFRA_NODE_INSTANCE_TYPE: Standard_D16s_v3
+      COMPUTE_NODE_TYPE: Standard_D8s_v5
+      OPENSHIFT_INFRA_NODE_INSTANCE_TYPE: Standard_D16s_v5
```

Comment:
Update Infra nodes to `Standard_E4s_v5`
Reference: [URL](https://docs.google.com/presentation/d/1OMtiM6S0yYCcE-sUeSNYfbSyZe8EtjQ2FGu-hqLdul4/edit#slide=id.gc52980a47d_1_0)